src/MicrosoftTeams.ts

```diff
@@ -588,6 +602,38 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    printCapabilityEnabled = true;
+  }
+
+  /**
+   * Registers a handler for print.
+   * default print handler
+   */
+  export function print(): void {
+    if (printCapabilityEnabled) {
+      ensureInitialized();
```

Comment:
Not sure if this is needed. If we only register the print handler when enablePrintCapability is called then all we need to do is call ensureInitialized() in there.

Reply:
thanks done.